app/models/aggregated_facility_review.rb

```diff
@@ -5,4 +5,21 @@ class AggregatedFacilityReview < ApplicationRecord
 
   belongs_to :facility
   belongs_to :space
+
+  def tooltip
+    case experience
+    when 'unknown'
+      'Usikkert. Ingen har spurt!'
+    when 'impossible'
+      'Umulig. De har ikke.'
+    when 'unlikely'
+      'Usannsynlig. Ingen eller få har fått lov'
+    when 'maybe'
+      'Kanskje! Ikke alle får lov'
+    when 'likely'
+      'Sannsynlig! Andre har fått lov!'
+    else
+      'Vet ikke'
+    end
```

Comment:
This could be a good place to use i18n to do the translation

Reply:
I think i8n might be overkill, or at least be something that we can add later on? 

There is no foreseeable future where the app will have non-Norwegian content. At least not while it's LNU hosted and owned, so translating the UI won't make much sense. 

That said, if some other country wants to use and contribute to the code base (e.g. the swedish, german, etc. variant of LNU), then i8n becomes relevant and we will have to translate everything from default seeds to hard coded strings all around the app.